integration-tests/src/main/java/io/druid/testing/ConfigFileConfigProvider.java

```diff
@@ -33,18 +32,21 @@
 public class ConfigFileConfigProvider implements IntegrationTestingConfigProvider
 {
   private final static Logger LOG = new Logger(ConfigFileConfigProvider.class);
-  private String routerHost = "";
-  private String brokerHost = "";
-  private String historicalHost = "";
-  private String coordinatorHost = "";
-  private String indexerHost = "";
-  private String middleManagerHost = "";
-  private String zookeeperHosts = "";        // comma-separated list of host:port
-  private String kafkaHost = "";
+  private String routerUrl;
+  private String brokerUrl;
+  private String historicalUrl;
+  private String coordinatorUrl;
+  private String indexerUrl;
+  private String middleManagerHost;
+  private String zookeeperHosts;        // comma-separated list of host:port
+  private String kafkaHost;
   private Map<String, String> props = null;
+  private String username;
```

Comment:
In future when druid supports auth natively, this property can be used for that, so I would prefer calling it username instead of prefixing it with gateway.